Agilefreaks/weatherios
# Feature/request location permissions

## Description

Use case
As a user I should be able to allow the app the get my current location so that I can eventually see the weather in my location

Acceptance criteria
Users should be prompted for the location access

## Related

https://trello.com/c/zN40271A

## Documentation

https://mobileinvader.com/corelocation-in-swiftui-mvvm-unit-tests/

## Link to demo

![Simulator Screen Shot - iPhone 12 - 2020-12-08 at 10 39 08](https://user-images.githubusercontent.com/15888842/101459969-9e836200-3941-11eb-8d3f-9555ec28d9bb.png)

## Steps to Test or Reproduce

1. Run app on device
2. You should see the request location permission pop-up
3. Accept permissions
4. You should see on the home view your latitude and longitude
